tests/test_detect.py

```
@@ -62,6 +62,33 @@ def test_reacnetgen(self, reacnetgen):
+        frame_cells = [np.eye(3) * 5.0, np.eye(3) * 10.0]
+
+        for step, cell in enumerate(frame_cells):
+            lattice = " ".join(str(value) for value in cell.ravel())
+            lines = (
+                "1\n",
+                f'Lattice="{lattice}" Properties=species:S:1:pos:R:3\n',
+                "H 0.0 0.0 0.0\n",
+            )
+            detector._readstepfunc((step, lines))
+
+        for call, expected_cell in zip(get_bonds.call_args_list, frame_cells):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)